### ISSUE TITLE
Solitaires with Russian deck

### ISSUE THREAD

**Issue by schw0reismus** [shlomif/PySolFC#286]
[Russian deck](https://en.wikipedia.org/wiki/Russian_playing_cards#Deck) is a regular card deck but without cards from 2 to 5 (thus consisting of 36 cards instead of 52). Is it possible for you to implement solitaires analogical to these with 52 cards?

For example, "Russian" Yukon would use 6 rows of cards instead of 7 and 4 open cards in each row (except the first) instead of 5. Klondike-like solitaires would use 6 rows instead of 7 as well.

**Comment by joeraz** [comment]
Wouldn't be that difficult - there already are a number of stripped deck games, though most of the current ones use the cards from 7-Ace or 9-Ace, and that same logic can work for 6-Ace with some minor modifications.  Unless you wanted "Russian" cards to be a separate type of deck, which would be more difficult.

Though we'd probably have to come up with a different name for "Russian Yukon" to avoid confusion with the existing "Russian Solitaire" game.

**Comment by schw0reismus** [comment]
Thanks for your answer. I can only say that have no evidence or information about the existence of such a game, not even saying about its name. Klondike with 36 cards may well exist though. Most websites in Russian say that solitaires are played with one or more 52-card decks, not 36.

**Comment by joeraz** [comment]
Right.  Let me know what you find out - I know a few stripped deck games that go both ways with the aces, or just use them as unique cards.

I think many of the alternate deck games were invented by previous PySol developers as variants of existing games adapted to the different deck - most of them were added before I was part of the project, but I did something similar when I added Hex Yukon.
